packages/amplify-graphql-api-construct-tests/src/__tests__/graphql-schemas/restricted-field-auth/ddb-only/graphql/schema.json

```diff
@@ -0,0 +1,7775 @@
+{
```

Comment:
Looks these are codegen generated files. Are we sure we want to check-in these files? Should we generate at runtime instead?